File: src/main/java/cloud/api/adress/AddressPoll.java
package cloud.api.adress;

import java.util.ArrayList;
import java.util.List;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-03-25
 *
 */

public class AddressPoll {
	
	private List<AddressRange> addresses = new ArrayList<AddressRange>();
	
	public AddressPoll() {
		
	}

	public List<AddressRange> getList() {
		return addresses;
	}

	public void setList(List<AddressRange> list) {
		this.addresses = list;
	}
	
}

File: src/main/java/cloud/api/address/AddressPoll.scala
package cloud.api.address
import java.util.{List => JList}
import reflect.BeanProperty

/**
 * @author Amir Moulavi
 * @date 2011-07-16
 *
 */
class AddressPoll {
  @BeanProperty var addresses:JList[AddressRange] = _
}